modules/xfem/src/efa/EFAElement3D.C

```diff
@@ -1037,15 +1041,34 @@ EFAElement3D::createChild(const std::set<EFAElement *> & CrackTipElements,
           center += cut_plane_points[i];
         center /= cut_plane_points.size();
 
+        for (unsigned int i = 0; i < cut_plane_points.size(); ++i)
+        {
+          int iplus1 = i < cut_plane_points.size() - 1 ? i + 1 : 0;
+          int iplus2 = i + 2;
+          if (iplus2 + 2 - cut_plane_points.size() >= 0)
+            iplus2 = iplus2 + 2 - cut_plane_points.size();
+
+          EFAPoint center(0.0, 0.0, 0.0);
+
+          center = cut_plane_points[i] + cut_plane_points[iplus1] + cut_plane_points[iplus2];
+          center /= 3;
+
+          orig_v.push_back((center));
+        }
+
         for (unsigned int i = 0; i < cut_plane_points.size(); ++i)
         {
           unsigned int iplus1 = i < cut_plane_points.size() - 1 ? i + 1 : 0;
-          EFAPoint ray1 = cut_plane_points[i] - center;
-          EFAPoint ray2 = cut_plane_points[iplus1] - center;
+          EFAPoint ray1 = cut_plane_points[i] - orig;
```

Comment:
My intuition is that it would be better to make the rays go between the current point and the two neighbors on either side. Something like this:
```
EFAPoint ray1 = cut_plane_points[i-1] -cut_plane_points[i];
EFAPoint ray2 = cut_plane_points[i+1] -cut_plane_points[i];
```
You would need to adjust the indices for when they go outside the size of cut_plane_points, and also check for the case where those rays are collinear, but this seems like it would give a more meaningful nodal normal.